hydra-cardano-api/src/Hydra/Cardano/Api/TxOut.hs

```diff
@@ -7,11 +7,11 @@ import Hydra.Cardano.Api.TxOutValue (mkTxOutValue)
 import Cardano.Api.UTxO qualified as UTxO
 import Cardano.Ledger.Api qualified as Ledger
 import Cardano.Ledger.Babbage.TxInfo qualified as Ledger
+import Cardano.Ledger.BaseTypes qualified as Ledger
```

Comment:
Should: not need this as I see you re-export `Network` still in `Hydra.Cardano.Api`

Reply:
This module does not depend on `Hydra.Cardano.Api`.